pkg/gardenlet/bootstrap/util/util.go

```diff
@@ -38,6 +43,42 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+const (
+	// GardenerSeedBootstrapper is a constant for the gardener seed bootstrapper name.
+	GardenerSeedBootstrapper = "gardener.cloud:system:seed-bootstrapper"
+	// DefaultSeedName is the default seed name in case the gardenlet config.SeedConfig is not set
+	DefaultSeedName = "<ambiguous>"
```

Comment:
This already exists here: https://github.com/gardener/gardener/blob/1c2c8eef3f19596f833c0bbc6b70f1196c0bd2fb/pkg/apis/core/v1beta1/constants/types_constants.go#L342